client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaLocationAndDocumentReference.vue

```diff
@@ -0,0 +1,365 @@
+<template>
+  <fieldset data-dp-validate="statementLocationAndDocumentReference">
+    <legend
+      id="locationAndDocuments"
+      class="mb-3 color-text-muted font-normal">
+      {{ Translator.trans( hasPermission('field_statement_polygon') ? 'location.and.document.reference' : 'document.reference') }}
+    </legend>
+
+    <!-- Location reference -->
+    <div
+      v-if="hasPermission('field_statement_polygon')"
+      class="font-semibold mb-1">
+      {{ Translator.trans('location') }}
+    </div>
+
+    <div class="space-y-4">
+      <div v-if="hasPermission('field_statement_polygon')">
+        <template v-if="statement.attributes.polygon">
+          <dp-button
+            :aria-label="Translator.trans('location.reference_view')"
+            :text="Translator.trans('map.view')"
+            variant="outline"
+            @click="toggleLocationModal" />
+          <dp-map-modal
+            ref="locationReferenceModal"
+            :procedure-id="procedureId" />
+        </template>
+        <template v-else>
+          -
+        </template>
+      </div>
+
+      <!-- Document reference -->
+      <div
+        v-if="editable"
+        class="grid grid-cols-1 gap-x-4 md:grid-cols-2">
+        <dp-select
+          data-cy="statement:elementSelect"
+          :label="{
+          text: Translator.trans('plandocument')
+        }"
+          :options="elementsOptions"
+          v-model="selectedElementId"
+          @select="handleSelect" />
+
+        <dp-select
+          v-if="paragraphOptions.length > 0"
+          data-cy="statement:paragraphSelect"
+          :label="{
+          text: Translator.trans('paragraph')
+        }"
```

Comment:
nit: indentation

Reply:
Thanks, done in 399dc66488c1e3ddc58f84a5a37db35fffca676d